src/java/org/apache/cassandra/metrics/KeyspaceMetrics.java

```diff
@@ -218,6 +230,13 @@ public KeyspaceMetrics(final Keyspace ks)
         // create histograms for TableMetrics to replicate updates to
         sstablesPerReadHistogram = createKeyspaceHistogram("SSTablesPerReadHistogram", true);
         tombstoneScannedHistogram = createKeyspaceHistogram("TombstoneScannedHistogram", false);
+        flushTime = createKeyspaceHistogram("FlushTime", false);
+        storageAttachedIndexRebuildTime = createKeyspaceHistogram("StorageAttachedIndexRebuildTime", false);
+        storageAttachedIndexWritingTimeForIndexBuild = createKeyspaceHistogram("StorageAttachedIndexWritingTimeIndexBuild", false);
+        storageAttachedIndexWritingTimeForCompaction = createKeyspaceHistogram("StorageAttachedIndexWritingTimeCompaction", false);
+        storageAttachedIndexWritingTimeForFlush = createKeyspaceHistogram("StorageAttachedIndexWritingTimeFlush", false);
+        storageAttachedIndexWritingTimeForOther = createKeyspaceHistogram("StorageAttachedIndexWritingTimeOther", false);
```

Comment:
the metrics names are not consistent between `KeyspaceMetrics` and `TableMetrics`

Reply:
good catch, fixed